CONTRIBUTING.md

```diff
@@ -355,7 +355,7 @@ You will need basic `git` proficiency to be able to contribute to
 manual. Type `git --help` in a shell and enjoy. If you prefer books, [Pro
 Git](https://git-scm.com/book/en/v2) is a very good reference.
 
-Follow these steps to start contributing ([supported Python versions](https://github.com/huggingface/diffusers/blob/main/setup.py#L265)):
+Follow these steps to start contributing ([supported Python versions](https://github.com/huggingface/diffusers/blob/main/setup.py#L270)):
```

Comment:
Actually, this should ideally be a permalink.

Reply:
Yeah, it makes more sense. I updated the link to a permalink. Please check now. Thanks!

Reply:
It will be only needed to update this permalink when supported Python versions are changed. Make sense :+1:.